README.md

```diff
@@ -1 +1,68 @@
-# nodejs-boilerplate
\ No newline at end of file
+![commercetools-icon](https://github.com/commercetools/press-kit/blob/master/PNG/72DPI/CT%20logo%20horizontal%20RGB%2072dpi.png?raw=true)
+
+# Node.js boilerplate
+[![Travis Build Status][travis-icon]][travis]
+[![Codecov Coverage Status][codecov-icon]][codecov]
+[![David Dependencies Status][david-icon]][david]
+[![David devDependencies Status][david-dev-icon]][david-dev]
+
+Start your next [Node.js commercetools](http://commercetools.github.io/nodejs-tools/) project in seconds.
+
+Table of Contents
+=================
+
+  * [Node.js boilerplate](#nodejs-boilerplate)
+    * [Features](#features)
+    * [Quick Start](#quick-start)
+    * [Requirements](#requirements)
+    * [Usage](#usage)
+    * [Development](#development)
+    * [Guidelines](#guidelines)
+
+## Features
+- Babel: use new JavaScript features while keeping compatibility.
+- TravisCI: config and setup included.
+- GreenKeeper: keep your dependencies up-to-date.
+- Guidelines: includes a contributing document
+- Readme: includes a readme template
+
+## Quick Start
+1. Clone this repo using `git clone --depth=1 https://github.com/commercetools/nodejs-boilerplate.git`
+2. Run `rm -r .git && npm run install` to clean the git repo and install dependencies.
+3. ???
+
+## Requirements
+Make sure you have installed all of the following prerequisites on your development machine:
+  * Git - [Download & Install Git](https://git-scm.com/downloads). OSX and Linux machines typically have this already installed.
+  * Node.js - [Download & Install Node.js](https://nodejs.org/en/download/) and the npm package manager. If you encounter any problems, you can also use this [GitHub Gist](https://gist.github.com/isaacs/579814) to instalÂl Node.js.
+
+## Usage
+  * Clone this repository
+
+  ```bash
+  $ git clone https://github.com/commercetools/nodejs-boilerplate.git nodejs-boilerplate
+  ```
+  * Run this command
+
+  ```
+  $ cd nodejs-boilerplate && npm install
+  ```
+  * Update the package.json and readme
+
+Now you're ready for liftoff!
+
```

Comment:
True, I merged the two readme's we had and wanted to keep both for reference. Just put them together. ;)